web/src/compositions/useBuild.ts

```diff
@@ -97,5 +97,15 @@ export default (build: Ref<Build | undefined>) => {
     return build.value?.ref;
   });
 
-  return { since, duration, message, prettyRef };
+  const started = computed(() => {
+    if (!build.value) {
+      return undefined;
+    }
+
+    const start = build.value.created_at || 0;
+
+    // sv-SE is in format YYYY-MM-DD HH:m:s
+    return new Date(start * 1000).toLocaleString('sv-SE');
```

Comment:
From my understanding, the started at time in MS is UTC, and the conversions should automatically use the proper time zone. I can test this once I get back from school.

Reply:
Timezone is properly selected!

Reply:
I would prefer to show it in the users local time format instead of using `sv-SE` for all users here. I will have a short test as it seems to not work out of the box on my browser for example.

Reply:
follow up pull ?!?

Reply:
I prefer sv-SE as it is the least ambiguous date format (`YYYY-mm-dd HH:m:s`)

Reply:
well I can understand both ... but in the end I think we have to respect what the user do have chosen on there system ...

Reply:
I'll see if I can put something together to add a user setting for it that defaults to the browser's setting in another PR.